src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRulesModule.java

```diff
@@ -300,6 +300,15 @@ public static class BuildGraveyardOptions extends OptionsBase {
         metadataTags = {OptionMetadataTag.INCOMPATIBLE_CHANGE},
         help = "When enabled java_common.compile only accepts JavaPluginInfo for plugins.")
     public boolean requireJavaPluginInfo;
+
+    @Option(
```

Comment:
```suggestion
    // This flag can be deleted after some references to it are removed
    // from some Google-internal config files.
    // Grep for `build_transitive_python_runfiles` in the Google codebase.
    @Option(
```

Reply:
I assume that would the procedure for removing anything in the graveyard.

Reply:
Yeah. I mean, sometimes we lag but that's the Platonic ideal :)